dal/dal.go

```diff
@@ -292,21 +296,33 @@ func (d *Dal) ClearCheckReference(memberID, keyName string) error {
 
 // Remove all key refs under individual member config dir
 func (d *Dal) ClearCheckReferences(memberID string) error {
-	_, err := d.KeysAPI.Delete(
-		context.Background(),
-		d.Prefix+"/cluster/members/"+memberID+"/config/",
-		&client.DeleteOptions{
-			Recursive: true,
-			Dir:       false,
-		},
-	)
+	// A recursive delete removes all child entries AND the dir itself; this is
```

Comment:
I have no idea how this was not noticed until just now. I noticed this race only because I tuned down the check stat dumping.